src/parser/warrior/fury/modules/talents/FuriousSlash.js

```diff
@@ -0,0 +1,121 @@
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import HIT_TYPES from 'game/HIT_TYPES';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import SpellLink from 'common/SpellLink';
+import { formatDuration, formatPercentage } from 'common/format';
+import React from 'react';
+import StatisticBox from 'interface/others/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+
+const MAX_FURIOUS_SLASH_STACKS = 3;
+
+class FuriousSlash extends Analyzer {
```

Comment:
I could be misunderstanding the logic behind this module, but as it's tracking a buff is there are reason it's using onCast events instead of listening to buff update events?

Reply:
It's tracking the buff uptime per stacks, and those stacks are updated by casting Furious Slash